src/main/java/ch/njol/skript/expressions/ExprLastAttacker.java

```diff
@@ -42,26 +45,21 @@
 	static {
 		register(ExprLastAttacker.class, Object.class, "last attacker", "entity");
 	}
+	@Nullable
+	ExprAttacker attackerExpr;
+	
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+		this.attackerExpr = new ExprAttacker();
```

Comment:
could you remove the `this.` here? your other file doesn't have it and i checked a couple expressions and it looks like most of them don't either